lib/beaker/options/command_line_parser.rb

```diff
@@ -98,6 +98,12 @@ def initialize
             @cmd_options[:timeout] = value
           end
 
+          opts.on '-r RUNNER', '--runner RUNNER',
+                  'Specify which test runner to use',
+                  'supported runners: beaker, minitest' do |value|
```

Comment:
That's a good point :eyes: :sparkles:  -- it will not do anything useful as of the moment. We actually should probably drop the ", minitest" from here and only add it when we want to make minitest available for public use.